core/src/main/resources/logback.xml

```diff
@@ -77,11 +77,20 @@
     </else>
   </if>
 
+  <!-- Configure the Sentry appender, overriding the logging threshold to the WARN level -->
+  <appender name="Sentry" class="com.getsentry.raven.logback.SentryAppender">
```

Comment:
Does this do anything for Mr & Mrs Average User running Cromwell on their laptops?